ikarus/controlroutines/controlinfos.hh

```diff
@@ -19,10 +19,25 @@ namespace Ikarus {
  */
 struct ControlInformation
 {
-  bool success{false}; ///< Flag indicating the success of the control.
-  std::vector<Ikarus::NonLinearSolverInformation>
-      solverInfos{};      ///< Vector containing information from nonlinear solvers.
-  int totalIterations{0}; ///< Total number of iterations performed.
+  bool success{false};                                   ///< Flag indicating the success of the control.
+  std::vector<NonLinearSolverInformation> solverInfos{}; ///< Vector containing information from nonlinear solvers.
+  int totalIterations{-1};                               ///< Total number of iterations performed.
+};
+
+/**
+ * \struct ControlLoggerInformation
```

Comment:
These things are not `ControlLoggerInformation` they are actually information of the control routine.

Maybe we should refactor the naming. Anyway the observers now have a `message` which is a enum and independent of the message, the observers always receive this struct. 

Therefore, maybe, if we want to keep the message notion, we can call this `ControlLoggerInformation` --> `ControlState` ?